packages/client/playwright/components/blotter.component.ts

```diff
@@ -57,31 +59,45 @@ export default class BlotterComponent extends BaseComponent {
     const tradeField = firstRowFields[0]
     return tradeField
   }
-  
+
+  public async latestTradeId () {
+    
```

Comment:
To follow convention we aim to use action verb for functions, I suggest `getLatestTradeId`